svd-encoder/src/config.rs

```diff
@@ -166,6 +166,44 @@ impl FromStr for FieldBitRangeFormat {
     }
 }
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum Sorting {
+    Offset,
+    OffsetReversed,
+    Name,
+}
+
```

Comment:
```suggestion
#[derive(Clone, Copy, Debug, PartialEq, Eq)]
#[non_exhaustive]
/// Apply a kind of sorting
pub enum Sorting {
    /// Sort by offsets
    Offset,
    /// Same as [`Sorting::Offset`], but reversed
    OffsetReversed,
    /// Sort by name
    Name,
}

```

Reply:
It would be neat also to have a 

```
for<'a> Fn(/*adress:*/ i32, /* name: */ &'a str, ... /*more?*/) -> Ordering
```

to implement alternative sorting schemes, but not really needed

Reply:
Interesting suggestion but I would leave it for someone other.

Reply:
It looks like this will require Config to depends 3 generics.